src/AppContainer.tsx

```diff
@@ -44,6 +45,9 @@ const AppContainer = (): React.ReactElement => {
           ))}
           {NETWORK === "ropsten" && (
             <FeatureFlag name="MAGIC_DEMO">
+              <Route path="/demo/verify" exact>
+                <DemoVerifyPage />
+              </Route>
```

Comment:
im wondering why we didnt put all the routes in the routes file, then pass in another key value to indicate whether to render a `ProtectedRoute` or not. I think this way was introduced when @ryanseee worked on it.

@ryanseee u have any comments about it?

Reply:
no, i introduced it. no particular reason, out of convenience lo cos needa feature flag + network now 😬